src/unix/notbsd/linux/other/mod.rs

```diff
@@ -206,6 +449,378 @@ pub const RUSAGE_CHILDREN: ::c_int = -1;
 pub const ST_RELATIME: ::c_ulong = 4096;
 pub const NI_MAXHOST: ::socklen_t = 1025;
 
+// (asm-generic/ioctl.h)
+pub const _IOC_NRBITS: u8 = 8;
+pub const _IOC_TYPEBITS: u8 = 8;
+pub const _IOC_SIZEBITS: u8 = 14;
+pub const _IOC_DIRBITS: u8 = 2;
+pub const _IOC_NRMASK: ::c_uint = (1 << _IOC_NRBITS) - 1;
+pub const _IOC_TYPEMASK: ::c_uint = (1 << _IOC_TYPEBITS) - 1;
+pub const _IOC_SIZEMASK: ::c_uint = (1 << _IOC_SIZEBITS) - 1;
+pub const _IOC_DIRMASK: ::c_uint = (1 << _IOC_DIRBITS) - 1;
+pub const _IOC_NRSHIFT: u8 = 0;
+pub const _IOC_TYPESHIFT: u8 = _IOC_NRSHIFT + _IOC_NRBITS;
+pub const _IOC_SIZESHIFT: u8 = _IOC_TYPESHIFT + _IOC_TYPEBITS;
+pub const _IOC_DIRSHIFT: u8 = _IOC_SIZESHIFT + _IOC_SIZEBITS;
+
+pub const _IOC_NONE: ::c_uint = 0;
+pub const _IOC_WRITE: ::c_uint = 1;
+pub const _IOC_READ: ::c_uint = 2;
+
+macro_rules! _IOC {
```

Comment:
For now we're actually avoiding these kinds of macros in the repo, it's fine to just inline all values as bit values.